job_creator.php

```diff
@@ -394,7 +402,7 @@ private function createPhpunitJobs(
                     'phpunit' => true,
                     'phpunit_suite' => $suite,
                 ]);
-            } elseif ($this->getCmsMajor() === '5') {
+            } elseif (in_array($this->getCmsMajor(), ['5', '6'])) {
```

Comment:
Why is this a condition at all? Why not just `else`? It's either 4 or it's something we want to do this with, right?

Reply:
Updated